### DISCUSSION TITLE
One or more errors occurred. (500 - An unhandled error occurred while serving the request (token: 81c0b3d07e6a4f6db59916197a1f9385).)

### DISCUSSION THREAD

**Discussion by kraoravooru** [datalust/seq-tickets#1245] [General]
Here i s my query
string query = $"select @Arrived, @Properties.Metrics as Metrics,@Properties.ReportType as ReportType," +
                                      $"@Properties.GeoMxVersion as GeoMxVersion from stream where Tags.ID like 'Diagnostics' ci and ReportType like '%RunHistory%'";

var connection = new SeqConnection(_seqSettings.ServerUrl,_seqSettings.ApiKey );
var result1 = Task.Run(()=> connection.Data.QueryAsync(query)).Result;

**Comment by nblumhardt** [comment]
Hi @kraoravooru - thanks for getting in touch!

This is very likely to be related to the ongoing issue we're investigating on your server (either because of the frequent disk space exhaustion, or the balance of machine resources vs. retained data volume).

I can loop back here and check this one out separately, if the changes we're discussing on the other thread don't resolve it. (The detailed error information matching the "token" will also be included in the diagnostic report we mentioned.) Fingers crossed this will be resolved along with the other problems, though.

Best regards,
Nick
